hw/top_earlgrey/data/autogen/top_earlgrey.gen.hjson

```diff
@@ -11647,6 +11647,9 @@
     pattgen
     otp_ctrl
     lc_ctrl
+    pwrmgr_aon
+    rstmgr_aon
+    clkmgr_aon
```

Comment:
Curious that the module name gets an "_aon" suffix (also in line 11799 and below). Is this an hjson quirk, just because the instance is named u_pwrmgr_aon (for example).

Reply:
I believe this is because we name the Hjson module instances like that:

https://github.com/lowRISC/opentitan/blob/b5467711be2a69eec570e7106378b5b648e0d605/hw/top_earlgrey/data/top_earlgrey.hjson#L405-L434

The `u_` prefix is prepended to that name in the SV template:

https://github.com/lowRISC/opentitan/blob/b5467711be2a69eec570e7106378b5b648e0d605/util/topgen/templates/toplevel.sv.tpl#L657

Reply:
I see, so those are not really SV module names, but some label hjson uses for them.

Reply:
yes exactly. the SV module names are derived from those label names.